gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/validator/CseValidRequestValidator.java

```diff
@@ -20,7 +20,7 @@
 @Component
 public class CseValidRequestValidator {
 
-    public void validateCseValidRequest(CseValidRequest cseValidRequest, Boolean isExportCornerActive) throws CseValidRequestValidatorException {
+    public void checkAllFilesExist(CseValidRequest cseValidRequest, boolean franceImporting) throws CseValidRequestValidatorException {
```

Comment:
franceImportingFromItaly ? I think it is good to keep the same point of view in all the code, compared to what you chose in the TTimestampWrapper, keep the same global convention. Either countryImportingFromItaly or countryExportingToItaly.